libs/vkd3d/vkd3d_private.h

```diff
@@ -618,8 +623,32 @@ struct d3d12_descriptor_heap
 
 HRESULT d3d12_descriptor_heap_create(struct d3d12_device *device,
         const D3D12_DESCRIPTOR_HEAP_DESC *desc, struct d3d12_descriptor_heap **descriptor_heap) DECLSPEC_HIDDEN;
+void d3d12_descriptor_heap_cleanup(struct d3d12_descriptor_heap *descriptor_heap) DECLSPEC_HIDDEN;
 struct d3d12_descriptor_heap *unsafe_impl_from_ID3D12DescriptorHeap(ID3D12DescriptorHeap *iface) DECLSPEC_HIDDEN;
 
+static inline unsigned int d3d12_descriptor_heap_sampler_set_index()
+{
+    return 0;
+}
+
+static inline unsigned int d3d12_descriptor_heap_cbv_set_index()
+{
+    return 0;
```

Comment:
Is this because we haven't really bothered hooking up CBV bindless yet?

Reply:
These are intended to be somewhat hard-coded (but therefore, very fast) functions to give us access to a descriptor set stored with a descriptor heap. The exact numbers don't really matter. This wouldn't change with bindless CBV.